src/test/scala/mesosphere/marathon/api/v2/json/EnrichedTaskWritesTest.scala

```diff
@@ -77,17 +78,20 @@ class EnrichedTaskWritesTest extends UnitTest {
         s"""
         |{
         |  "appId": "${f.runSpecId}",
+        |  "healthCheckResults" : [],
         |  "id": "${f.taskWithoutIp.task.taskId.idString}",
+        |  "ipAddresses" : [],
         |  "host": "agent1.mesos",
         |  "state": "TASK_STAGING",
         |  "ports": [],
-        |  "startedAt": null,
+        |  "servicePorts" : [],
```

Comment:
If I interpret this change correctly, it means that previously, there was always a field `"startedAt"`, even if the value was null. That field does not seem to be the case after this change. Given that tooling might expect this field to be present, wouldn't this be a breaking change?